CodeGeneration/Sources/generate-swiftsyntax/templates/swiftsyntaxbuilder/SwiftSyntaxBuilderCompatibilityFile.swift

```diff
@@ -0,0 +1,45 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+import SwiftSyntaxBuilder
+import SyntaxSupport
+import Utils
+
+let swiftSyntaxBuilderCompatibilityFile = try! SourceFileSyntax(leadingTrivia: copyrightHeader) {
+  DeclSyntax("import SwiftSyntax")
+
+  DeclSyntax(
+    """
+    @available(*, deprecated, renamed: "ImportPathBuilder")
+    public typealias AccessPathBuilder = ImportPathBuilder
+    """
+  )
```

Comment:
Similar here, I would prefer to just keep this static content out of CodeGeneration. It just makes it easier to modify.